exe/launcher/Main.hs

```diff
@@ -74,15 +81,17 @@ main = do
     when (args `isPresent` (shortOption 'h')) $ help cli
     when (args `isPresent` (longOption "version")) $ putStrLn getVersion
 
-    network <- args `parseArg` longOption "network"
+    let stateDir = args `getArg` (longOption "state-dir")
+    let network = fromMaybe "testnet" $ args `getArg` (longOption "network")
```

Comment:
Just a small point, but parentheses are not actually needed in the above two lines.